### DISCUSSION TITLE
Don't know where to report these two things

### DISCUSSION THREAD

**Discussion by kalzEOS** [elementary/triage#124] [Q&A] [answered]
I am trying to report/file/request or whatever two things, but I don't know where to do so as I couldn't find a "proper" place for them. 
1. Where do I report the very low frame rate that I have on my laptop testing EOS 6 beta. Since there is no Nvidia on the beta, I am running solely on intel's UHD 620 integrated graphics card. I have a 4k screen on my laptop and the intel/nvidia hybrid graphics. The UI behaves like the frame rate is 20 FPS when I move the windows or do the overview. So, I wanted report it and see if the team has any fixes coming to the stable release to improve the performance. Or maybe it is just a kerenl thing, I don't know.

2. The contrast (or maybe the font) is a little too faint (or the font is too thin or not too black, I don't know). I am having a hard time keeping my eyes on the screen when the theme is light. I feel like it is a little hard to read the font on the light background. I had to increase the size of the font to 11 to be able to read things comfortably. I am trying to address this to the team, but my English is failing me as I am not able to correctly convey what I am trying to report, and I also don't know where to do so.
If anyone could chime in and help, I would really appreciate it. I am a huge fan of EOS and I love running it on all of my machines. Also, I want to participate (by testing on my main machine and report things on here) in the project to help make it even better.
Thank you

**Comment by danirabbit** [comment, marked as answer]
Hey, sorry for taking quite a while to reply :sweat_smile: 

We've done quite a lot of work making the window manager silky smooth for OS 7, but the repository for reporting those kinds of issues would be https://github.com/elementary/gala

For look and feel issues such as text constrast, those would need to be reported against the elementary stylesheet at https://github.com/elementary/stylesheet

**Comment by danirabbit** [comment]
Closing as a duplicate of https://github.com/orgs/elementary/discussions/621
